tools/federation/src/test/java/org/eclipse/rdf4j/federated/PropertyPathTests.java

```diff
@@ -105,20 +110,12 @@ public void testPropertyPath_ExclusiveGroup() throws Exception {
 	}
 
 	@Test
-	public void testPropertyPath_ExclusivePath() throws Exception {
+	public void testPropertyPath_Combination() throws Exception {
```

Comment:
this test from the query plan was actually not correct in the general case. Note that I no longer compare the query plan, but the results